src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
```

Comment:
The same issue with the formatting of the `link` tag attributes. Each attribute should start on a new line with a 2-space indentation. The closing bracket should be on the same level as the opening one.